.semaphore/semaphore.yml

```diff
@@ -141,10 +137,6 @@ blocks:
       commands:
       - cd typha
       - make ci EXCEPT=k8sfv-test
-    - name: "Typha: felix k8s-st tests"
```

Comment:
This is just a duplicate of the tests already run in the Felix section - straight up running the same tests twice.